src/com/javarush/task/task07/task0706/Solution.java

```diff
@@ -0,0 +1,83 @@
+package com.javarush.task.task07.task0706;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/*
+Улицы и дома
+*/
+
+public class Solution {
+    private static int even = 0;
+    private static int odd = 0;
+
+    private static String text = null;
+
+    public static void main(String[] args) throws IOException {
+        //напишите тут ваш код
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        int[] street = new int[15];
+        for (int i = 0; i < street.length; i++) {
+            street[i] = Integer.parseInt(reader.readLine());
+        }
+        mmm(street);
+        print();
+
+
+    }
+
+    public static void mmm(int[] argss) {
+        for (int i = 0; i < argss.length; i = i + 2) {
+            even = even + argss[i];
+        }
+        for (int i = 1; i < argss.length; i = i + 2) {
+            odd = odd + argss[i];
+        }
+
+    }
+
+    public static void print() {
+        if (even > odd) {
+            text = "четными";
+        } else {
+            text = "нечетными";
+        }
+        System.out.printf("В домах с %s номерами проживает больше жителей.", text);
```

Comment:
Ну или пойти дальше и использовать тернарный оператор:
```suggestion
        System.out.printf("В домах с %s номерами проживает больше жителей.", even > odd ? "четными" : "нечетными");
```